cli_tools/gce_vm_image_import/importer/inflater.go

```diff
@@ -194,23 +195,20 @@ type persistentDisk struct {
 	isUEFICompatible bool
 }
 
-type inflationInfo struct {
+type shadowTestFields struct {
 	// Below fields are for shadow API inflation metrics
 	checksum      string
 	inflationTime time.Duration
 	inflationType string
 }
 
-func createInflater(args ImportArguments, computeClient daisyCompute.Client, storageClient storage.Client, inspector imagefile.Inspector) (inflater, error) {
-	di, err := createDaisyInflater(args, inspector)
-	if err != nil {
-		return nil, err
-	}
-
-	return di, nil
+func createInflater(args ImportArguments, computeClient daisyCompute.Client, storageClient storage.Client,
```

Comment:
Quite a few unused args. Is this to support API inflater creation?

Reply:
Yeah; @dntczdx's VMDK project uses these.